pxblue/react-native-component-library
# Update components to handle device rotation

For PXBLUE-2206
Fixes #76 .


#### Changes proposed in this Pull Request:
- Add new hook for getting the correct height constants based on device orientation
- Updated landscape styles for Header and Drawer


#### Screenshots:
![image](https://user-images.githubusercontent.com/29152776/126535020-9156027e-d3c2-4846-9d60-c7e931cf56d9.png)
![image](https://user-images.githubusercontent.com/29152776/126535050-f77847f4-91f0-4140-a083-bf9d76657c90.png)


#### To Test:
- clone the repo
- `yarn start:showcase`
- play to your heart's content

Make sure to test out things like: 
- opening and closing the drawer
- rotating while the drawer is open
- expanding and collapsing the header
- rotating all the way around
- changing the RTL mode and rotating
- etc.
